### PR TITLE
feat(terms and conditions): added terms and condtions component

### PR DESCRIPTION
**Pull request by abhishek-mishra019**
Signed-off-by: Abhishek Mishra <am1426620@gmail.com>

### Functionality:
added terms and conditions component

### Solution:
added terms and conditions component for the footer and setup app route for the component, then added content in the html file.

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
in footer click "Terms
Screenshot from 2021-12-14 00-14-40
"

### CODE REVIEW

**Review by viv9k — APPROVED** [review]
Approved

**Review by Pendyala-thanvi — COMMENTED** [review]
Resolve merge conflicts

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -26,6 +26,11 @@ import { CopyrightComponent } from './body/copyright/copyright.component';
 import { Patch8Component } from './patches/patch8/patch8.component';
 import { PrivacyComponent} from './body/privacy/privacy.component';
 import { Patch7Component } from './patches/patch7/patch7.component';
+<<<<<<< HEAD
```
Comment (Pendyala-thanvi) [inline src/app/app-routing.module.ts:29]: remove merge markers

File: src/app/app.module.ts [hunk 2]
```diff
@@ -102,7 +102,11 @@ import { EditSkillsComponent } from './body/profile/edit-skills/edit-skills.comp
 import { NotificationCenterComponent } from './header/notification-center/notification-center.component';
 import { Patch7Component } from './patches/patch7/patch7.component';
 import { CopyrightComponent } from './body/copyright/copyright.component';
+<<<<<<< HEAD
```
Comment (Pendyala-thanvi) [inline src/app/app.module.ts:105]: remove merge markers here too.

File: src/app/app.module.ts [hunk 3]
```diff
@@ -188,7 +192,16 @@ import { Patch8Component } from './patches/patch8/patch8.component';
     EditEducationComponent,
     EditWorkexperienceComponent,
     EditProjectsComponent,
+<<<<<<< HEAD
```
Comment (Pendyala-thanvi) [inline src/app/app.module.ts:195]: same here

**Pull request merged**